easybuild/easyconfigs/q/Qhull/Qhull-2015.2-intel-2016a.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'CMakeMake'
+
+name = 'Qhull'
+version = '2015.2'
+
+homepage = 'http://www.qhull.org'
+description = """
+Qhull computes the convex hull, Delaunay triangulation, Voronoi diagram, halfspace intersection about a point,
+furthest-site Delaunay triangulation, and furthest-site Voronoi diagram. The source code runs in 2-d, 3-d, 4-d,
+and higher dimensions. Qhull implements the Quickhull algorithm for computing the convex hull.
+"""
+
+toolchain = {'name': 'intel', 'version': '2016a'}
+
+#recently checkged the Unix version numbering see https://github.com/Homebrew/homebrew-science/issues/3353
```

Comment:
@boegel: Its not so clean for me, the windows version kept its download name (2015.2) and version (2015.2), the unix one changed its download name (2015-src-7.2.0), but it kept its base directory as `qhull-2015.2`. So what I think that version numbering is a mess, and they do not explain that in the `Changes.txt`.  
To be more precise, in the past they used `qhull_VERSION2` (2015.2), and internally `qhull_VERSION` (7.2.0) now it is a mixture  of the two in download names.

Honestly the whole issue does not worth this amount of writing, I am not even sure why the developers changed the download name.